src/main/java/christmas/constant/Badge.java

```diff
@@ -0,0 +1,42 @@
+package christmas.constant;
+
+public enum Badge {
+    /*
+        5천 원 이상: 별
+        1만 원 이상: 트리
+        2만 원 이상: 산타
+     */
+    NONE("없음", 0),
+    STAR("별", -5_000),
+    TREE("트리", -10_000),
+    SANTA("산타", -20_000);
+
+    private final String name;
+    private final int condition;
+
+    Badge(String name, int condition) {
+        this.name = name;
+        this.condition = condition;
+    }
+
+    public int getCondition() {
+        return condition;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public static Badge getBadgeByTotalBenefitAmount(int totalBenefitAmount) {
+        if (totalBenefitAmount <= SANTA.getCondition()) {
+            return SANTA;
+        }
+        if (totalBenefitAmount <= TREE.getCondition()) {
+            return TREE;
+        }
+        if (totalBenefitAmount <= STAR.getCondition()) {
+            return STAR;
+        }
+        return NONE;
```

Comment:
해당 부분은 `enum`의 선언 순서만 조금 바꿔 준다면 아래와 같이 스트림으로 리팩토링이 가능한데요.
아래 코드의 장점은 새로운 뱃지가 추가되더라도 로직에는 변경이 필요 없다는 점입니다!
```suggestion
        Arrays.stream(Badge.values())
                .filter(badge -> badge.condition >= totalBenefitAmount)
                .findFirst()
                .orElse(NONE);
```